paymentsheet/src/main/java/com/stripe/android/link/ui/LinkContent.kt

```diff
@@ -137,20 +118,23 @@ private fun Screens(
     changeEmail: () -> Unit,
     initialDestination: LinkScreen,
 ) {
-    NavHost(
-        modifier = Modifier.fillMaxSize(),
+    LinkNavHost(
         navController = navController,
-        startDestination = initialDestination.route,
+        initialDestination = initialDestination,
     ) {
         composable(LinkScreen.Loading.route) {
-            Loader()
+            LinkLoadingScreen()
         }
 
         composable(LinkScreen.SignUp.route) {
-            SignUpRoute(
-                navigateAndClearStack = navigateAndClearStack,
-                moveToWeb = moveToWeb
-            )
+            // Keep height fixed to reduce animations caused by IME toggling on both
+            // this screen and Verification screen.
+            MinScreenHeightBox {
```

Comment:
:eyes: Demo of the problem:

https://github.com/user-attachments/assets/c3eb4994-2259-4f02-aa07-6f587fcf5819